UITableView_pageSetting/ViewController.swift

```diff
@@ -8,12 +8,146 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    let idCell = "MyCell"
+    @IBOutlet weak var tableView: UITableView!
+    
+    var items: [String] = ["we", "heart", "Swift"]
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        tableView.dataSource = self
+        self.tableView.backgroundColor = UIColor.lightGray
+        self.tableView.sectionHeaderHeight = 10
     }
+    
+}
+
+struct Cell {
+    var title: String?
+}
 
 
+class CellApi {
+    static func getData() -> [[Cell]] {
+        let section1 = [Cell(title: "Авиарежим"),
+                        Cell(title: "Wi-Fi"),
+                        Cell(title: "Bluetooth"),
+                        Cell(title: "Сотовая связь"),
+                        Cell(title: "Режим модема"),
+                        Cell(title: "VPN")]
+
+        return [section1]
+    }
+}
```

Comment:
Структуры хватит да и зачем так сложно массив в массиве + нейминг не удачный
```suggestion
struct CellApi {
    static func getData() -> [Cell] {
        let section1 = [Cell(title: "Авиарежим"),
                        Cell(title: "Wi-Fi"),
                        Cell(title: "Bluetooth"),
                        Cell(title: "Сотовая связь"),
                        Cell(title: "Режим модема"),
                        Cell(title: "VPN")]

        return section1
    }
}
```

Reply:
исправил